x-pack/elastic-agent/pkg/agent/application/upgrade/service.go

```diff
@@ -200,8 +200,8 @@ func (p *noopPidProvider) Name() string { return "noop" }
 
 func (p *noopPidProvider) PID(ctx context.Context) (int, error) { return 0, nil }
 
-func invokeCmd() *exec.Cmd {
-	homeExePath := filepath.Join(paths.Home(), agentName)
+func invokeCmd(topPath string) *exec.Cmd {
```

Comment:
I saw the name topPath in the past? What does it exactly mean?

Reply:
with the structure we have topPath is the top of agent directory structure e.g. `/etc/Elastic/Agent`
we also use a homePath which points to a home of a certain agent version e.g `/etc/Elastic/Agent/data/elastic-agent-abc123/`